src/components/AllTimeStats/AllTimeStats.test.js

```diff
@@ -0,0 +1,12 @@
+import {
+  render, screen,
+} from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import AllTimeStats from './AllTimeStats';
+import store from '../../Redux/storeSetUp';
+
+it('rendering navbar', () => {
+  render(<Router><Provider store={store}><AllTimeStats /></Provider></Router>);
```

Comment:
[OPTIONAL] It would be better to mock the access to the API so that your tests don't send actual requests.
And also you may need to mock the connection to the Redux Store.